packages/react-charting/src/utilities/utilities.ts

```diff
@@ -68,22 +69,22 @@ export interface IAxisData {
 export interface IMargins {
   /**
    * left margin for the chart.
-   * @default 40
+   * @default 80
```

Comment:
Dont change default margins. Adjust them if the title is defined.

Reply:
done

Reply:
This is still not fixed